ens/main.py

```diff
@@ -75,14 +76,19 @@ def fromWeb3(cls, web3, addr=None):
         '''
         return cls(web3.manager.providers, addr=addr)
 
-    def address(self, name):
+    def address(self, name, guess_tld=True):
```

Comment:
I'm beginning to be of the opinion that we should just **require** the full name and not support guessing the TLD.  This is like a browser letting you type `google` and it taking you magically to `google.com`.

Reply:
>  This is like a browser letting you type google and it taking you magically to google.com

I actually use ctrl-enter a lot in browsers, so I might have pitched doing that in a browser. heh.

But yeah, I can see the argument for it. Autofilling .eth made a little more sense when it was essentially the only TLD. That's still basically true, but not for much longer.